simplified-app-shared/src/main/java/org/nypl/simplified/app/reader/ReaderReadiumJavaScriptAPI.java

```diff
@@ -180,53 +180,23 @@ private void evaluateWithResult(
     final ReaderSettingsType r)
   {
     try {
-      final ReaderColorScheme cs = r.getColorScheme();
-      final String color = NullCheck.notNull(
-        String.format("#%06x", cs.getForegroundColor() & 0xffffff));
-      final String background = NullCheck.notNull(
-        String.format("#%06x", cs.getBackgroundColor() & 0xffffff));
-
-      final JSONObject decls = new JSONObject();
-      decls.put("color", color);
-      decls.put("backgroundColor", background);
+      String fontSelected = "";
 
       switch (r.getFontFamily()) {
         case READER_FONT_SANS_SERIF: {
-          decls.put("font-family", "sans-serif");
+          fontSelected = "sans-serif";
           break;
         }
         case READER_FONT_OPEN_DYSLEXIC: {
-          /**
-           * This is defined as a custom CSS font family inside
-           * OpenDyslexic.css, which is referenced from the initially
-           * loaded reader.html file.
-           */
-
-          decls.put("font-family", "OpenDyslexic3");
+          fontSelected = "OpenDyslexic3";
           break;
         }
         case READER_FONT_SERIF: {
-          decls.put("font-family", "serif");
+          fontSelected = "serif";
           break;
         }
       }
 
-      final JSONObject o = new JSONObject();
-      o.put("selector", "*");
-      o.put("declarations", decls);
-
-      final JSONArray styles = new JSONArray();
-      styles.put(o);
-
-      final StringBuilder script = new StringBuilder(256);
-      script.append("ReadiumSDK.reader.setBookStyles(");
-      script.append(styles);
-      script.append("); ");
-      script.append("document.body.style.backgroundColor = \"");
```

Comment:
I noticed that this *does* update the background color, but even when it's removed, the background color _still_ changes. So it seems that there's multiple pieces of code that does the same thing to the reader?

I also tried updating the foreground color using `document.body.style.color` but this does not work (for the white text on black background). The `setBookStyles` function on the ReadiumSDK does not actually update the background or foreground colors.